bin/rusty_vault.rs

```diff
@@ -0,0 +1,23 @@
+use std::process::ExitCode;
+
+use clap::{Command};
+use rusty_vault::cli;
+
+fn main() -> ExitCode {
+    let mut app = Command::new("rusty_vault")
+        .version(rusty_vault::VERSION)
+        .help_expected(true)
+        .disable_colored_help(false)
+        .max_term_width(100)
+        .about("A self-controlled, secure, and high-performance open-source software for key management that is compatible with Vault.");
```

Comment:
Delete "self controlled" and "key management". Could be something like:

~~~
A secure and high performance secret management software that is compatible with Hashicorp Vault
~~~

Reply:
done